2018-02-02T03:41:53.757Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '21' in collection with ID 'Asset:org.decentralized.energy.network.DDoS' does not exist)
    at /home/batista/project/project/go/src/github.com/hyperledger/ddos-defense-based-blockchain/angular-app/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-02-02T03:41:53.758Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '21' in collection with ID 'Asset:org.decentralized.energy.network.DDoS' does not exist)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '21' in collection with ID 'Asset:org.decentralized.energy.network.DDoS' does not exist)\n\n    at _initializeChannel.then.then.then.then.catch (/home/batista/project/project/go/src/github.com/hyperledger/ddos-defense-based-blockchain/angular-app/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-02-02T03:41:53.893Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '21' in collection with ID 'Asset:org.decentralized.energy.network.DDoS' does not exist)
    at /home/batista/project/project/go/src/github.com/hyperledger/ddos-defense-based-blockchain/angular-app/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-02-02T03:41:53.893Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '21' in collection with ID 'Asset:org.decentralized.energy.network.DDoS' does not exist)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '21' in collection with ID 'Asset:org.decentralized.energy.network.DDoS' does not exist)\n\n    at _initializeChannel.then.then.then.then.catch (/home/batista/project/project/go/src/github.com/hyperledger/ddos-defense-based-blockchain/angular-app/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
